cli/cmd/tapRunner.go

```diff
@@ -219,40 +267,21 @@ func readValidationRules(file string) (string, error) {
 	return string(newContent), nil
 }
 
-func createMizuResources(ctx context.Context, kubernetesProvider *kubernetes.Provider, serializedValidationRules string, serializedContract string, serializedMizuConfig string) error {
+func createMizuResources(ctx context.Context, cancel context.CancelFunc, kubernetesProvider *kubernetes.Provider, serializedValidationRules string, serializedContract string, serializedMizuConfig string) error {
+	var err error
```

Comment:
why do you need to declare the err var here?

Reply:
the line below `state.mizuServiceAccountExists, err = createRBACIfNecessary(ctx, kubernetesProvider)` sets an existing var's value and declares err, so I have to declare err before

I moved it next to that line so its clearer